types/token-introspection/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for token-introspection 3.0
+// Project: https://github.com/schibsted/node-token-introspection
+// Definitions by: Daniel Rose <https://github.com/DanielRose>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { RSA_JWK } from 'pem-jwk';
+import { RequestInit, Response } from 'node-fetch';
+import { JwtPayload } from 'jsonwebtoken';
+
+import customErrors = require('./errors');
+
+declare function fetch(url: string, options: RequestInit): Promise<Response>;
```

Comment:
```suggestion
type  Fetch = (url: string, options: RequestInit) => Promise<Response>;
```
should work same IMO, without declaring fake symbol, which doen't exist in source